admin/preprints/views.py

```diff
@@ -81,6 +88,13 @@ def get_context_data(self, **kwargs):
         kwargs.update({'message': kwargs.get('message')})
         return super(PreprintView, self).get_context_data(**kwargs)
 
+    def update_subjects_for_provider(self, request, old_provider, new_provider):
+        subject_problems = self.object.map_subjects_between_providers(old_provider, new_provider, Auth(request.user))
```

Comment:
OK, I think we should pass in auth=None here.  Then our model can bypass permission checks if auth is None.  See `Node.set_privacy` for an example of this.